src/diffusers/loaders.py

```diff
@@ -2507,6 +2520,19 @@ def enable_lora(self):
         if hasattr(self, "text_encoder_2"):
             self.enable_lora_for_text_encoder(self.text_encoder_2)
 
+    def delete_lora(self, adapter_name):
+        if not USE_PEFT_BACKEND:
+            raise ValueError("PEFT backend is required for this method.")
+
+        # Enable unet adapters
```

Comment:
We're not enabling anything here no?

Reply:
But we have enable_lora, disable_lora, unload_lora_weights, load_lora_weights